packages/nc-gui/lang/en.json

```diff
@@ -304,6 +304,7 @@
     "importExcel": "Import Excel",
     "downloadCSV": "Download as CSV",
     "uploadCSV": "Upload CSV",
+    "downloadAttachments": "Download Attachments",
```

Comment:
To make it consistent, I'd suggest you to apply the same change to all files [here](https://github.com/nocodb/nocodb/tree/e251ed6f95e528b27fdee4d41ec5a19b01a74df9/packages/nc-gui/lang) with the same position. We'll update the master spreadsheet after that.